roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -43,6 +43,14 @@ ExecStart=/usr/bin/rkt run \
 {% if '/var/lib/kubelet' not in kubelet_flexvolumes_plugins_dir %}
         --volume flexvolumes,kind=host,source={{ kubelet_flexvolumes_plugins_dir }},readOnly=false \
 {% endif %}
+{% if local_volume_provisioner_enabled %}
```

Comment:
Should we add a `ExecPre mkdir {{ local_volume_provisioner_mount_dir }}` to this to ensure dir exists?

Reply:
Not necessary since it's in kubernetes preinstall task now.